### ISSUE TITLE
training loss

### ISSUE THREAD

**Issue by merlinarer** [OpenMOSS/Language-Model-SAEs#64]
I run with qwen1.5B and the loss seems really small compared with saelens:
![捕获](https://github.com/user-attachments/assets/d1a1d467-ab38-4516-9d1c-d2ae84a4d1f3)
Could you share the loss curve of llama scope?

**Comment by Hzfinfdu** [comment]
I need some time to figure out how to make these wandb dashboards available. Here are some examples of Llama Scope training curve:
![image](https://github.com/user-attachments/assets/bae919b6-7c22-47ea-8ca5-0ca1b0727399)

Having loss with orders of magnitudes smaller than SAELens is okay. Maybe bc we use a reconstruction loss with the mean of errors of each element rather than sum, or somewhere else. Anyways the absolute value of loss does not matter much since it will be compensated by learning rate and lead to (nearly) the same training dynamics if configured properly.

I do not see problem in your config. If explained variance starts from ~0.9 and decreases till it plateaus at~0.7 when k decreases to 50, like shown above. That means you are getting everything right.

Hope this helps with your question

**Comment by Hzfinfdu** [comment]
You are also welcomed to contact me via Wechat: Hzfinfdu0728
